composer.json

```diff
@@ -36,12 +36,15 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Symfony\\Cmf\\Bundle\\ContentBundle\\Tests\\": "tests/"
+            "Symfony\\Cmf\\Bundle\\ContentBundle\\Tests\\": "tests/",
+            "Symfony\\Cmf\\Bundle\\ContentBundle\\Tests\\App\\": "tests/Resources/app"
```

Comment:
we could also put the kernel into tests/App/AppKernel.php. isn't that the recommended way of doing it?

Reply:
good point

Reply:
normally a new Kernle lives in src/Kernel.php and got a App\Kernel as FQCN.

Reply:
in new flex apps

Reply:
then lets put our kernel in Tests\App and give it whatever namespace is right there.

Reply:
`*\Tests\App\AppKernel` feels like the namespace you request me, right? but that is the one used on every bundle now. Is it the folder, only? no clue if i can change that one as whole testing is a little bit tricky with directories. Moving kernel out of that directory bingo is a first step.

Reply:
i don't understand the problem. if our kernel FQN is Symfony\Cmf\Bundle\ContentBundle\Tests\App\Kernel (or AppKernel, whatever) then it could just live in `tests/App/Kernel.php` and we need no special autoloading rule. or we can make the FQN Symfony\Cmf\Bundle\ContentBundle\Tests\Fixtures\App\Kernel if we want to better namespace this.

Reply:
OK. i will do so.

Reply:
I like the `* Fixtures\App\Kernel` more, as we do have several test apps with documents and controllers, ...